### ISSUE TITLE
Tooltip a11y needs work

### ISSUE THREAD

**Issue by pixelbandito** [cision/rover-ui#305]
Misc a11y fixes to bring behavior in line with recommendations for tooltips here: https://knowbility.org/blog/2018/WCAG21-1413ContentHoverFocus 

Known issues:
* Content is rendered in an unintuitive place. 
* Focus / blur doesn't work like hovering.
* In click-to-open mode, the content isn't being read by screen readers.
* The user can't dismiss the tooltip from the keyboard in hover mode.
* The close button is an icon with no text representation.

**Comment by pixelbandito** [comment]
Fixed in [v2.6.0](https://github.com/cision/rover-ui/releases/tag/v2.6.0)
